### PR TITLE
[*] Add Retool Dashboard

### PR DESCRIPTION
**Pull request by KuhArt**

### CODE REVIEW

File: services/api-koa/src/logger.ts [hunk 1]
```diff
@@ -1,12 +1,20 @@
 import winston from 'winston';
 import config from 'config';
 
+const formatToPrettyJson = winston.format.printf(info => {
```
Comment (KuhArt) [inline services/api-koa/src/logger.ts:4]: Fix Object serializing. Previosly, `[Object object]` was returned

File: services/api-koa/src/resources/user/actions/remove.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import userService from 'resources/user/user.service';
+import { AppKoaContext, AppRouter } from 'types';
+
+type ValidatedData = never;
+
+async function handler(ctx: AppKoaContext<ValidatedData>) {
+  await userService.removeSoft({ _id: ctx.request.params?.id });
```
Comment (Oigen43) [inline services/api-koa/src/resources/user/actions/remove.ts:7]:
```suggestion
  await userService.removeSoft({ _id: ctx.request.params.id });
```
`id` always will be in params

File: services/api-koa/src/resources/user/actions/update.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+type ValidatedData = {
+  firstName: string,
+  lastName: string,
+  email: string
+};
+
+async function handler(ctx: AppKoaContext<ValidatedData>) {
+  const { firstName, lastName, email } = ctx.validatedData;
+  const updatedUser = await userService.update({ _id: ctx.request.params?.id }, () => ({ firstName, lastName, email }));
+  if (!updatedUser) {
```
Comment (Oigen43) [inline services/api-koa/src/resources/user/actions/update.ts:26]: let's add validator that user exists before handler

File: services/api-koa/src/resources/user/actions/remove.ts (hunk 2 above)
Comment (Oigen43) [reply services/api-koa/src/resources/user/actions/remove.ts:7]: let's add validator that user exists

**Review by Oigen43 — CHANGES_REQUESTED** [review]

**Comment by Oigen43** [comment]
Please return package-lock.json in koa-service 🥺

**Comment by Oigen43** [comment]
What is `misc` folder? Do we really need it?

File: services/api-koa/src/resources/user/actions/remove.ts (hunk 2 above)
Comment (KuhArt) [reply services/api-koa/src/resources/user/actions/remove.ts:15]: >id always will be in params

According to types error it is possible the params can be `undefined`

**Comment by Oigen43** [comment]
Let's fix `ctx.request.params?.id` TypeScript error

**Pull request merged**